0_run.R

```diff
@@ -0,0 +1,38 @@
+#Set the working directory to the path where this script is stored.
+setwd(dirname(rstudioapi::getActiveDocumentContext()$path))
+
+###################### 1_fetch
+source("1_fetch/src/input_functions.R")
+
+#Install/load packages
+packages<-c("dplyr", "readr", "stringr", "sbtools", "whisker")
+check.packages(packages)
+
+outPath1 <- "C://Projects/ds-pipelines-targets-1-course-static/1_fetch/out"
```

Comment:
This comment is similar to the comment above with one additional problem - I do not have a path that starts with `C://Projects/` on my work computer so I cannot run the code in `0_run.R` as written.

For the same reasons I mention above, you can get away with a local file path here
```suggestion
outPath1 <- "1_fetch/out"
```